dpctl-capi/include/dpctl_vector.h

```diff
@@ -41,6 +41,10 @@ DPCTL_C_EXTERN_C_BEGIN
     __dpctl_give DPCTL##EL##VectorRef DPCTL##EL##Vector_Create();              \
                                                                                \
     DPCTL_API                                                                  \
+    __dpctl_give DPCTL##EL##VectorRef DPCTL##EL##Vector_CreateFromArray(       \
+        size_t len, DPCTLSycl##EL##Ref *elems);                                \
```

Comment:
`*elems` should be marked `__dpctl_take`

Reply:
done